workspaces/sentry/plugins/sentry/src/api/annotations.ts

```diff
@@ -18,6 +18,12 @@ import { Entity } from '@backstage/catalog-model';
 
 export const SENTRY_PROJECT_SLUG_ANNOTATION = 'sentry.io/project-slug';
 
+export const hasSentryAnnotation = (entity: Entity) => {
```

Comment:
Should we use this instead?

https://github.com/backstage/community-plugins/blob/6cc64e66ae10474f02e0179584abcd483dafb268/workspaces/sentry/plugins/sentry/src/components/isSentryAvailable.ts#L24